docs/builder/manifest.md

```diff
@@ -0,0 +1,291 @@
+# Customizing Your Web Manifest
+
+Many capabilities of progressive web apps are exposed through your app's web manifest, and you can use the PWABuilder service to edit your manifest and enhance your app.
+
+
+To edit your manifest via the PWABuilder site:
+
+1. Get to your app's report card page. If you need guidance on this, vist the [Quick Start.](/builder/quick-start)
+
+2. In the `Manifest` subsection, click `Edit Your Manifest`.
+
+3. A modal with several tabs will open. Each tab will contain different settings for editing your manifest, along with brief tooltips on what each field means.
```

Comment:
```suggestion
3. A modal with several tabs will open. Each tab will contain different settings for editing your manifest, along with brief tooltips on what each member means.
```

Reply:
Same here... nit - manifest fields are called members. Not going to correct them all in the rest of the md file. Please correct as you see fit.